prov/usnic/src/usdf_fabric.c

```diff
@@ -839,17 +853,25 @@ usdf_fabric_open(struct fi_fabric_attr *fattrp, struct fid_fabric **fabric,
 	dp = __usdf_devinfo;
 	for (d = 0; d < dp->uu_num_devs; ++d) {
 		dep = &dp->uu_info[d];
+		if (dep->ue_dev)
+			USDF_DEBUG("XXX DJG dep->ue_dattr.uda_devname=%s\n",
+				dep->ue_dattr.uda_devname);
+		else
+			USDF_DEBUG("XXX DJG dep->ue_dev=%p\n", dep->ue_dev);
```

Comment:
Were these XXX DJG's left in by accident?  Seems somewhat weird that there's a case where dep->ue_dev would be NULL...?